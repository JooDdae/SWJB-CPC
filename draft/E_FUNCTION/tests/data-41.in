34
4 12 3 31 2 -1
33 14 34 9 6 27 -1
26 16 28 23 22 32 9 -1
-1
29 -1
15 4 10 30 9 2 8 6 5 27 28 17 29 18 -1
9 29 3 -1
15 23 25 8 6 34 5 19 31 17 33 32 22 7 11 13 30 24 14 16 29 26 18 21 10 2 -1
9 5 3 32 29 22 7 2 6 27 19 11 34 33 8 25 28 31 -1
15 32 14 9 27 23 17 11 2 30 31 8 4 33 5 12 22 10 6 34 16 20 29 7 19 21 18 -1
13 11 3 30 18 16 7 24 32 -1
24 21 29 -1
23 11 13 31 16 12 17 7 4 25 14 3 24 30 33 15 -1
27 17 14 20 5 24 25 10 11 16 23 33 28 4 31 9 30 34 21 26 8 29 32 22 2 7 18 19 15 13 6 -1
4 26 9 17 15 7 32 27 3 21 33 13 6 18 31 10 23 24 11 12 -1
5 10 3 11 31 21 16 29 26 28 30 8 24 25 13 19 7 22 17 23 18 12 -1
31 18 7 20 8 6 11 33 14 3 5 27 26 4 25 30 16 17 12 28 21 19 9 34 10 15 -1
10 20 15 24 4 27 2 3 26 -1
25 12 13 5 24 -1
28 27 33 20 29 3 9 16 31 32 18 24 7 19 15 23 17 22 4 2 30 -1
20 23 25 3 2 16 4 12 33 7 21 26 24 31 29 30 9 34 10 22 17 -1
27 10 32 -1
-1
15 29 11 18 5 24 6 17 34 7 33 30 10 32 -1
12 29 18 3 13 21 27 2 25 5 24 10 34 20 16 30 11 9 22 28 7 4 6 17 32 31 19 14 15 -1
34 27 15 33 21 26 28 30 24 14 23 -1
11 3 25 33 5 15 6 17 12 -1
30 18 29 20 33 9 13 10 17 11 19 6 5 25 26 4 31 34 21 22 14 8 3 12 -1
31 16 15 22 8 27 23 17 20 -1
24 22 34 14 15 26 30 6 5 25 2 3 16 17 11 27 7 10 18 33 4 12 29 13 9 32 8 28 20 31 23 19 -1
29 33 3 24 10 23 25 22 6 9 2 8 30 32 26 13 18 4 5 20 34 21 14 16 11 12 15 7 31 -1
33 26 16 15 -1
28 30 6 17 21 32 2 -1
3 8 9 22 4 33 20 5 21 18 6 13 25 11 17 7 28 2 -1
