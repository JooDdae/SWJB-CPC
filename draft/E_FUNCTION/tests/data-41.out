YES
3 26 34 